arduino/cores/packageindex/testdata/package_Zoubworld_index.json

```diff
@@ -11,61 +11,61 @@
       },
       "platforms": [
         
-{

```

Comment:
Why this section is changed? :thinking:

Reply:
It probably converted spaces into tabs or viceversa